test/runtests.jl

```diff
@@ -13,7 +13,7 @@ end
 @testset "DiffRules" begin
 @testset "check rules" begin
 
-non_numeric_arg_functions = [(:Base, :rem2pi, 2), (:Base, :ifelse, 3)]
+non_numeric_arg_functions = [(:Base, :rem2pi, 2), (:Base, :ifelse, 3), (:Base, :mod, 2)]
```

Comment:
This is misleading, the arguments are not non-numeric.

Reply:
Ok. Then maybe we could have a separate list? Or rename `non_numeric_arg_functions` to `discontinuous_functions`?

Reply:
I don't think it has to be handled in a more special way than the different unary functions. A branch in the binary tests or shifting values there should be sufficient.